sop-core/src/Data/SOP/NP.hs

```diff
@@ -10,6 +15,8 @@ module Data.SOP.NP
   , pure_POP
   , cpure_NP
   , cpure_POP
+  , (++)
```

Comment:
This is actually exporting the term-level `(++)` from the `Prelude`, not the type-level `(++)` that you defined. I think you intended to export `type (++)` instead.

Reply:
Thanks. Fixed! I'm not dedicated to the names btw.